mk/PX4/ROMFS/init.d/rc.APM

```diff
@@ -82,24 +82,6 @@ then
     mkblctrl -mkmode x -d /dev/pwm_output
 fi
 
-if mtd start /fs/mtd
```

Comment:
I actually don't remember if I did. If I did it was a pixhawk 2, not 1. I'll check again.